### DISCUSSION TITLE
How to access Analytics data?

### DISCUSSION THREAD

**Discussion by joergrech** [redouane59/twittered#405] [Q&A]
Hi, 
I'm not sure if the analytics data is available via the Twitter API but I could really use it. What I mean is the data from https://analytics.twitter.com/user/<TWITTER_NAME>/tweets for one month, including, number of tweets, retweets, likes, replies, engagement, etc.

The only way I currently see is to use a Web-Scraper, Login, and download the Export data for a month.

**Comment by redouane59** [comment]
Hi @joergrech , this information is not available in the Twitter API I used. I don't know if other endpoints exists, maybe related to https://developer.twitter.com/en/docs/twitter-ads-api ?
